Invalid timestamps after slicing `Kymo` (#179)
lumicks/pylake: changelog.md

#### Bug fixes

> * Fixed bug in `Kymo` which resulted in erroneous timestamps and line times after slicing the Kymograph by time. The reason for this was imprecision in the timestamp reconstruction that occurred when slicing the data. This in turn led to an erroneous reconstruction of the timestamps. **As a result, any downstream analysis that relies on the time axis of a Kymograph post-slicing cannot be trusted (MSD analysis, plotting, tracking, etc) in pylake v0.8.2 and v0.9.0.** Only those two versions are affected. Note that regular image reconstruction was not affected.

Reviewer: We might add at the end something like:
"Timestamps of un-sliced `Kymo`s constructed directly from the .h5 files are also unaffected"
just to really nail down the point that this bug is only when you slice

> * Fixed bug in `Kymo` which resulted erroneous timestamps and line times after slicing the Kymograph by time. The reason for this was an imprecision in the timestamp reconstruction that occurred when slicing the data. This in turn which led to an erroneous reconstruction of the timestamps. **As a result, any downstream analysis that relies on the time axis of a Kymograph post-slicing cannot be trusted (MSD analysis, plotting, tracking, etc).** This error was introduced in Pylake v0.8.2 and both v0.8.2 and v0.9.0 are affected. Note that the image reconstruction was not affected.

Reviewer 2: ```suggestion
* Fixed bug in `Kymo` which resulted in erroneous timestamps and line times after slicing the Kymograph by time. The reason for this was imprecision in the timestamp reconstruction that occurred when slicing the data. This in turn led to an erroneous reconstruction of the timestamps. **As a result, any downstream analysis that relies on the time axis of a Kymograph post-slicing cannot be trusted (MSD analysis, plotting, tracking, etc) in pylake v0.8.2 and v0.9.0.** Only those two versions are affected. Note that regular image reconstruction was not affected.
```
